brage-import/src/main/java/no/sikt/nva/brage/migration/mapper/PublicationContextMapper.java

```diff
@@ -100,11 +100,18 @@ public static PublicationContext buildPublicationContext(Record record)
         }
         if (isReaderOpinion(record)) {
             return buildPublicationContextWhenReaderOpinion();
+        }
+        if (isCristinRecord(record)) {
+            return null;
```

Comment:
So this here is just to prevent an error to be thrown?
I'd rather have an error "UnableToMapRecordError" to be thrown, than to make this lambda "fail" silently.